lib/Voxa-Opearlo.js

```diff
@@ -26,27 +26,25 @@ function register(skill, config) {
   skill.onAfterStateChanged((alexaEvent, reply, transition) => {
     OpearloAnalytics.registerVoiceEvent(
       alexaEvent.session.user.userId,
-      transition.to,
+      'Custom',
       alexaEvent.intent.name,
-      alexaEvent.intent.params);
+      _.merge({ to: transition.to }, alexaEvent.intent.params));
 
-    alexaEvent.opearloRegistered = true;
     debug(`${transition.to} registered`);
   });
 
   skill.onBeforeReplySent((alexaEvent, reply, transition) => new Promise((resolve, reject) => {
     OpearloAnalytics.recordAnalytics(
       alexaEvent.session.user.userId,
-      pluginConfig.opearloApiKey,
+      pluginConfig.apiKey,
       (result) => {
         debug(`recordAnalytics (state: ${transition.to}); result: ${result}`);
         resolve();
       });
   }));
 
   skill.onSessionEnded((alexaEvent) => {
-    OpearloAnalytics.registerVoiceEvent(alexaEvent.session.user.userId, 'die', 'SessionEndedRequest');
-    alexaEvent.opearloRegistered = true;
+    OpearloAnalytics.registerVoiceEvent(alexaEvent.session.user.userId, 'SessionEndedRequest', 'die');
```

Comment:
there's no `SessionEndedRequest` event type according to https://analytics.opearlo.com/setup-analytics/node/alexa-sdk